webview/src/shared/components/icons/Error.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
```

Comment:
We decided to remove the`AllIcons` enum that gave us the ability to view all available icons since it was duplicating the index file in the icons folder. We should either export this inside the index file or simply remove the index file.

Reply:
I think we have a couple of other direct imports. I'll track them down today and move all imports to come from the index file. Let me know if you'd like to go the other way.